src/include/OSL/oslquery.h

```diff
@@ -79,6 +79,10 @@ class OSLQUERYPUBLIC OSLQuery {
         Parameter () {}
         Parameter (const Parameter& src);
         Parameter (Parameter&& src);
+        // because a move constructor is defined, assignment operators will be implicitly deleted
+        // which is a problem for use in std::vectors
+        Parameter &operator=(const Parameter &) = default;
```

Comment:
Unfortunately, this is incorrect. The default assignment is wrong for the same reason we can't rely on the default copy constructor or mover.

It's my fault. I'll fix this for real and submit a patch.